code/index.html

```diff
@@ -58,7 +71,11 @@ <h1>Does the person have</h1>
         <h1 id="winOrLoseText"></h1>
         <button id="playAgain" class="filled-button">PLAY AGAIN</button>
       </div>
+      
     </section>
-    <script src="script.js"></script>
+    <script src="script.js"></script> 
   </body>
+  <footer class="footer">
```

Comment:
I like that you added a footer, which looks really nice :)

Reply:
Nice to add also the link to your Github/LinkedIn profile when your nick is clicked :D